src/utils/authorizations.ts

```diff
@@ -0,0 +1,97 @@
+    case isSbcFieldOfficeStaff(): return SbcFieldOfficeStaffRoles.includes(action)
+    default: return DefaultRoles.includes(action)
+  }
+}
+
+/**
+ * Whether the user is Business Registry Staff.
+ * Ultimately we won't need this function and we'll just check auth roles for everything.
+ */
+function isBusinessRegistryStaff (): boolean {
+  const account = useBcrosAccount()
+  return account.authRoles?.includes(AuthorizationRolesE.STAFF)
+}
+
+/**
+ * Whether the user is Maximus Staff.
+ * Ultimately we won't need this function and we'll just check auth roles for everything.
+ */
+function isMaximusStaff (): boolean {
+  const account = useBcrosAccount()
+  return account.authRoles?.includes(AuthorizationRolesE.MAXIMUS_STAFF)
+}
+
+/**
+ * Whether the user is Contact Centre Staff.
+ * Ultimately we won't need this function and we'll just check auth roles for everything.
+ */
+function isContactCentreStaff (): boolean {
+  const account = useBcrosAccount()
+  return account.authRoles?.includes(AuthorizationRolesE.CONTACT_CENTRE_STAFF)
+}
+
+/**
+ * Whether the user is SBC Field Office Staff.
+ * Ultimately we won't need this function and we'll just check auth roles for everything.
+ */
+function isSbcFieldOfficeStaff (): boolean {
+  const account = useBcrosAccount()
+  // return account.authRoles.includes(AuthorizationRolesE.SBC_STAFF) // *** TODO: uncomment this after #27536
+  return (account.currentAccount?.accountType === AccountTypeE.SBC_STAFF) // *** TODO: delete this after #27536
+}
+
+/**
+ * The roles if the user is Business Registry Staff.
+ * Ultimately we won't need this list and we'll just check auth roles for everything.
+ */
+const BusinessRegistryStaffRoles = [
+  AuthorizedActionsE.CONVERSION_FILING,
+  AuthorizedActionsE.CORRECTION_FILING,
+  AuthorizedActionsE.ENABLE_DOCUMENT_RECORDS,
+  AuthorizedActionsE.NO_CONTACT_INFO,
+  AuthorizedActionsE.STAFF_BREADCRUMBS,
+  AuthorizedActionsE.STAFF_FILING,
+  AuthorizedActionsE.STAFF_COMMENTS
+]
+
+/**
+ * The roles if the user is Maximus Staff.
+ * Ultimately we won't need this list and we'll just check auth roles for everything.
+ */
+const MaximusStaffRoles = []
+
+/**
+ * The roles if the user is Contact Centre Staff.
+ * Ultimately we won't need this list and we'll just check auth roles for everything.
+ */
+const ContactCentreStaffRoles = []
+
+/**
+ * The roles if the user is SBC Field Office Staff (aka SBC Staff Tier 2).
+ * Ultimately we won't need this list and we'll just check auth roles for everything.
+ */
+const SbcFieldOfficeStaffRoles = []
+
+/**
+ * The roles if the user is none of the other types.
+ * Ultimately we won't need this list and we'll just check auth roles for everything.
+ */
+const DefaultRoles = [
+  AuthorizedActionsE.CONVERSION_FILING,
+  AuthorizedActionsE.CORRECTION_FILING,
+  AuthorizedActionsE.ENABLE_DOCUMENT_RECORDS,
+  AuthorizedActionsE.SPECIAL_RESOLUTION_FILING
+]
```

Comment:
No, regular users cannot do conversions or corrections.

Reply:
However, regular users are authorized to do regular files -- see spreadsheet -- eg, AGM Extension, Consent for Amalgamation Out, etc, etc.

Reply:
I'm expecting to see a lot more actions that regular users can do here, like:
- ADDRESS_CHANGE
- AGM_EXTENSION_FILING
- AGM_CHG_LOCATION_FILING
- ALTERATION_FILING
- AMALGAMATION_FILING
- ANNUAL_REPORT_FILING
- CONSENT_AMALGAMATION_OUT_FILING
- CONSENT_CONTINUATION_OUT_FILING
- DELAY_DISSOLUTION_FILING
- SPECIAL_RESOLUTION_FILING
- VOLUNTARY_DISSOLUTION_FILING

(Double-check in case I missed any.)

(I see the Business Dashboard UI sheet is incorrect for End User/Client. Please update that, too.)

Reply:
Looks good. Thanks!

You didn't include DELAY_DISSOLUTION_FILING but that OK as it's not implemented yet 😆 

Resolving.